### PR TITLE
add support for marks

### PR DESCRIPTION
**Pull request by anakojm**
This PR adresses #7 
Let me now if you want me to change anything, I'm pretty new to python and contributing

### CODE REVIEW

**Comment by anakojm** [comment]
What should I do with [tests/test_wiki_links_processor.py](https://github.com/devidw/obsidian-to-hugo/pull/10/files#diff-aac5d1424ce6494dd5d8c687dacc9f6a07245d3cd1cc6f27ad9ff86a97339378)? Should I rename it to `test_processor.py` or should I split the tests in two files, one for wiki links, the other one for marks?

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ It takes care of the following steps:
 - Copies obsidian vault contents into hugo content directory (`.obsidian` gets removed immediately after copying)
 - Replaces obsidian wiki links (`[[wikilink]]`) with hugo shortcode links
   (`[wikilink]({{< ref "wikilink" >}})`)
+- replace obsidian marks (`==important==`) with html marks (`<mark>important</mark>`)
```
Comment (devidw) [inline README.md:27]:
```suggestion
- Replaces obsidian marks (`==important==`) with html marks (`<mark>important</mark>`)
```

File: tests/test_wiki_links_processor.py [hunk 2]
```diff
@@ -61,5 +62,30 @@ def test_replace_wiki_links(self):
         self.assertEqual(real_out, expected_out)
 
 
+class TestMarksProcessor(unittest.TestCase):
```
Comment (devidw) [inline tests/test_wiki_links_processor.py:65]:
```suggestion
class MarksProcessorTestCase(unittest.TestCase):
```

**Review by devidw — CHANGES_REQUESTED** [review]
Just a few little naming things

**Comment by devidw** [comment]
Awesome work @anakojm 

> What should I do with [tests/test_wiki_links_processor.py](https://github.com/devidw/obsidian-to-hugo/pull/10/files#diff-aac5d1424ce6494dd5d8c687dacc9f6a07245d3cd1cc6f27ad9ff86a97339378)? Should I rename it to test_processors.py or should I split the tests in two files, one for wiki links, the other one for marks?

I would suggest splitting it down into separate test cases in separate files

Just make sure the filename starts with `test_`, since this is the default pattern the built-in unit test package looks for

**Review by devidw — APPROVED** [review]

**Comment by devidw** [comment]
Thanks @anakojm 🎉

**Pull request merged**